BASE/Test/ServerTelemetryChannel.Test/TelemetryChannel.Nuget.Tests/TelemetryChannel.Nuget.Tests.csproj

```diff
@@ -12,21 +12,21 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net45;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
     <AssemblyName>TelemetryChannel.Nuget.Tests</AssemblyName>
     <DebugType>pdbonly</DebugType> 
     <DebugSymbols>true</DebugSymbols> 
   </PropertyGroup>
 
   <PropertyGroup Condition="$(OS) != 'Windows_NT'">
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
please remove netstandard1.3. Test projects are runnable as such, and its pointless to target netstandard. I dont know why it was targetting netstandard1.3 earlier! It should have been netcoreapp1.1!

Reply:
@cijothomas , we have a problem here. if we add netcoreapp1.1, we dont have a package for Microsoft.Web.Xdt available. Should i remove standard1.3 and netcoreapp1.1? What do u think?

Reply:
For .net core nugets, there are no XDT transformations at all. Its only for .netframework. - Hence this channel.nugets.tests should only target .NETFramework and not .net core.

Reply:
you can essentially abandon this PR as there is nothing to do for channel nuget tests for .net core at all!

Reply:
i see...ok then :)